sos/policies/redhat.py

```diff
@@ -39,7 +39,7 @@ class RedHatPolicy(LinuxPolicy):
     upload_directory = '/incoming'
     default_container_runtime = 'podman'
     sos_pkg_name = 'sos'
-    sos_bin_path = '/usr/sbin/sosreport'
+    sos_bin_path = '/usr/sbin'
```

Comment:
Shouldn't we appy the same also to https://github.com/sosreport/sos/blob/master/sos/policies/debian.py#L20 ?

Reply:
Nice catch! I actually just need to remove that line from Debian entirely, and leave it as the policy default.